lib/so-info.c

```diff
@@ -791,17 +832,11 @@ int so_info_lookup_cu_function_name(struct bt_dwarf_cu *cu, uint64_t addr,
 			goto error;
 		}
 
-		snprintf(offset_str, ADDR_STR_LEN, "+%#0" PRIx64,
-				addr - low_addr);
-		_func_name = malloc(strlen(die_name) + ADDR_STR_LEN);
-		if (!_func_name) {
+		ret = so_info_append_offset_str(die_name, low_addr, addr,
+						func_name);
```

Comment:
Both of these were already fixed at the time of your comment in a later commit. All of these are gonna get squashed prior to merging.